robot: {
    goal_pos_param: {
        x_goal: 0,
        y_goal: 0
    }
}